localgov_base.theme

```diff
@@ -43,22 +43,20 @@ function localgov_base_preprocess_page(&$variables) {
   // @see https://www.drupal.org/node/953034
   // @see https://www.drupal.org/forum/support/module-development-and-code-questions/2016-04-07/drupal-8-regions-with-and-empty#comment-12149518
   $active_theme = \Drupal::service('theme.manager')->getActiveTheme()->getName();
-  $system_region = system_region_list($active_theme, $show = REGIONS_ALL);
+  $regions = array_keys(system_region_list($active_theme, REGIONS_VISIBLE));
 
-  foreach ($system_region as $key => $value) {
-    $region = $key;
-    $has = 'has_' . $region;
+  $excluded_regions = [
+    'messages',
+    'disabled',
```

Comment:
Does disabled get returned with REGIONS_VISIBLE set on the request? Are there disabled not visible regions? What are they for?

Reply:
In a theme's info.yml file, there's an optional key called 'regions_hidden', which the docs describe as 'A list of inherited regions to remove.'. If you pass the REGIONS_ALL constant to system_region_list(), these get returned. If you pass REGIONS_VISIBLE, they don't.

So my understanding of this is that it's a mechanism to allow a theme to not inherit a region from its base theme, and we shouldn't try to do anything with regions that haven't been inherited.